libraries/views.py

```diff
@@ -370,6 +370,8 @@ def get_related(self, library_version, relation="maintainers", exclude_ids=None)
         for user in qs:
             if author_email := commit_authors.get(user.email.lower(), None):
                 user.commitauthor = author_email.author
+            elif user.display_name:
+                user.commitauthor = user
```

Comment:
This is a bit hacky. Would it work if we set the `display_name` in the `SimpleNamespace` object below to `user.display_name`?

Reply:
I tested this locally, and it does work.

```python
            if author_email := commit_authors.get(user.email.lower(), None):
                user.commitauthor = author_email.author
            else:
                user.commitauthor = SimpleNamespace(
                    github_profile_url="",
                    avatar_url="",
                    display_name=user.display_name,
                )